Bottom: 081d95eb5ac612b1f1136ba159a2ccb5716a75ee
Top:    e1efc0f3e7744ece5658ea9de4ae2e0621c8d195
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 19:40:47 +0100

hid/gtk: Make layer selector show layers in group ordering.

This means re-shuffling layers displayed so layers in the same group are
drawn adjacent to each other. This will allow rendering to show which
layers are grouped together.


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index ee79f1a..28e4b71 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -844,6 +844,8 @@ make_layer_buttons (GtkWidget *layersel)
   gchar *color_string;
   gboolean active = TRUE;
 
+  /* XXX: I want these in group order! */
+
   for (i = 0; i < max_copper_layer; ++i)
     {
       layer_process (&color_string, &text, &active, i);
